src/routes/MovieListPage/MovieListPage.jsx

```diff
@@ -0,0 +1,95 @@
+import { useState } from "react";
+import { Outlet, useLoaderData, useNavigate, useSearchParams } from "react-router-dom";
+
+import { GenreSelect } from '../../Components/GenreSelect/GenreSelect';
+import { SortControl } from '../../Components/SortControl/SortControl';
+import { MovieTile } from '../../Components/MovieTile/MovieTile';
```

Comment:
Good point! Remember it!

There're difficulties with aliases because of create-react-app package.
I fixed it for tests and storybook, but didn't have enough patience to fix it for app.